lib/response.ex

```diff
@@ -46,11 +47,18 @@ defmodule HTTPServer.Response do
     }
   end
 
+  def build_location_header(path) do
+    %{
+      "Location" => "http://#{@host}#{path}"
+    }
+  end
```

Comment:
adding a named function for all the hundreds of potential http headers will eventually get tedious. Have you considered the [builder pattern](https://refactoring.guru/design-patterns/builder)?

Reply:
The builder pattern refactor is getting pretty big, so I'm going to make it it's own PR!